RecoCTPPS/PixelLocal/plugins/CTPPSPixelClusterProducer.cc

```diff
@@ -0,0 +1,77 @@
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "RecoCTPPS/PixelLocal/interface/CTPPSPixelClusterProducer.h"
+
+CTPPSPixelClusterProducer::CTPPSPixelClusterProducer(const edm::ParameterSet& conf) :
+  param_(conf) ,
+  clusterizer_(conf){
+  
+  src_ = conf.getParameter<std::string>("label");
+  verbosity_ = conf.getUntrackedParameter<int> ("RPixVerbosity");
+	 
+  tokenCTPPSPixelDigi_ = consumes<edm::DetSetVector<CTPPSPixelDigi> >(edm::InputTag(src_));
+ 
+  produces<edm::DetSetVector<CTPPSPixelCluster> > ();
+  }
+
+CTPPSPixelClusterProducer::~CTPPSPixelClusterProducer() {
+
+}
+
+void CTPPSPixelClusterProducer::fillDescriptions(edm::ConfigurationDescriptions & descriptions){
+  edm::ParameterSetDescription desc;
+  desc.addUntracked<int>("RPixVerbosity",0);
+  desc.add<std::string>("label", "ctppsPixelDigis");
+  desc.add<int>("SeedADCThreshold",15);
+  desc.add<int>("ADCThreshold",10);
+  desc.add<double>("ElectronADCGain",135.0);
+  desc.add<int>("VCaltoElectronGain",50);
+  desc.add<int>("VCaltoElectronOffset",-411);
+  desc.add<bool>("doSingleCalibration",false); 
+ descriptions.add("clusterProd", desc);
```

Comment:
it's nice to have the fillDescriptions implemented.
If this is actually usable, the _cfi file should be removed.
The clusterProd_cfi file is already created by the framework and can be used directly
(not, however that the name does not follow the style of many other producers and should probably be ctppsPixelClusters instead).